influxql/INFLUXQL.md

```diff
@@ -653,15 +664,17 @@ SELECT mean(value) FROM cpu WHERE region = 'uswest' GROUP BY time(10m) fill(0);
 ```
 from_clause     = "FROM" measurements .
 
-group_by_clause = "GROUP BY" dimensions fill(<option>).
+group_by_clause = "GROUP BY" dimensions fill(fill_option).
+
+into_clause     = "INTO" measurement .
```

Comment:
This line is actually slightly incorrect in that it does not reflect the `":" measurement` rule that appears [here in the code](https://github.com/influxdb/influxdb/blob/418c8cd6f4dbafccf4484ca67af562fdbbae10e2/influxql/parser.go#L921-L928). That functionality doesn't seem to be documented anywhere else that I was able to quickly find.